src/components/ContactUs/ContactUs.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Footer from "components/Footer/Footer.js";
+import "./ContactUs.scss";
+export default function ContactUs() {
+  return (
+    <>
+      <div className="contact-us">
+        <div className="meet-our-team">MEET OUR TEAM</div>
+        <div className="members-img-container">
+          <div className="members-img-wrapper">
+            <img
+              id="contact-us-img"
+              alt="sehwan"
+              src={require("/Users/hongseoupyun/Desktop/hongseoup/sideProjects/compcar-ui/src/assets/Sehwan.jpg")}
```

Comment:
Since absolute path has been set in `jsconfig`, all src paths should be like "assets/Sehwan.jpg".

Reply:
same as `Navbar.js`